Pipfile

```diff
@@ -19,5 +19,4 @@ pytest-pep8 = "==1.0.6"
 pytest-mock = "==1.7.1"
 pytest-cov = "==2.5.1"
 pudb = "==2017.1.4"
-snakeviz = "==0.4.2"
```

Comment:
Would it be more effective to install this dependent on the python version, the same as for `typing`?